s3_storage_client.go

```diff
@@ -87,8 +87,11 @@ func (util *snowflakeS3Client) getFileHeader(meta *fileMetadata, filename string
 			}
 			meta.resStatus = errStatus
 			meta.lastError = err
-			return nil, fmt.Errorf("error while retrieving header")
+			return nil, fmt.Errorf("s3 api error while retrieving header")
```

Comment:
I think it would be better to leave it as is, some people may depend on this message.

Reply:
ok